app/models.py

```diff
@@ -260,6 +260,8 @@ class SupplierFramework(db.Model):
                              db.ForeignKey('frameworks.id'),
                              primary_key=True)
     declaration = db.Column(JSON)
+    on_framework = db.Column(db.Boolean, nullable=True)
```

Comment:
Yep that's right - if no application was made then this column will never get populated (though there could be a completed declaration if they didn't complete any services).

We might or might not want to clear out the "didn't apply" rows in future, but shouldn't be a problem in the meantime.